src/evo/cbtx.cpp

```diff
@@ -215,15 +215,15 @@ bool CalcCbTxMerkleRootQuorums(const CBlock& block, const CBlockIndex* pindexPre
             auto qcHash = ::SerializeHash(qc.commitment);
             const auto& llmq_params = llmq::GetLLMQParams(qc.commitment.llmqType);
             auto& v = qcHashes[llmq_params.type];
-            if (v.size() == llmq_params.signingActiveQuorumCount) {
+            if (v.size() == uint64_t(llmq_params.signingActiveQuorumCount)) {
```

Comment:
`size_t(...)` maybe? or make `signingActiveQuorumCount` `size_t` (same for other `LLMQParams` members, not sure why we use `int` for them...)